docs/versions.md

```diff
@@ -25,14 +25,13 @@ For example, the code might display version `2.3`.
 
 ## Use the REST API version supported by the server
 
-To always use the latest version of the REST API that is supported by the instance of Tableau Server you are connecting to, set the version using the `use_server_version()` method. You can do this when you create the server instance by specifying `use_server_version=True` as one of the arguments. 
```

Comment:
Both methods are still valid, we just didn't have an example for the first method. It would looks like this:
```
server = TSC.Server('http://SERVER_URL')
server.use_server_version()
```

Reply:
And there are reasons you might want to use the separate version rather than the argument version.  For instance, if you want to disable certificate checking, you have to do that after you create the server object, but before you make calls to the server (use_server_version makes a call to the server).

Reply:
got it - my earlier understanding was wrong, but I've rolled this back in now